src/components/Home.vue

```diff
@@ -4,22 +4,59 @@
       <p>{{ userName }}さんようこそ!!</p>
       <div class="button">
         <p>残高{{ wallet }}</p>
+        <input
+          type="button"
+          value="ログアウト"
+          @click="signOut()"
+          class="logout"
+        />
       </div>
     </div>
     <h1>ユーザ一覧</h1>
-    <p>ユーザ名</p>
+    <div>
+      <p>ユーザ名</p>
+      <table class="usernamelist">
+        <tr v-for="(newUser, index) in newUsers" :key="index" class="usertr">
+          <td class="tableuser">{{ newUser.userName }}</td>
+          <div>
+            <td>
+              <button class="walletButton" @click="() => openModal(index)">
```

Comment:
メソッド名だけで指定することができますね。
```suggestion
              <button class="walletButton" @click="openModal(index)">
```